src/main/java/io/reactivex/internal/operators/flowable/FlowableDebounceTimed.java

```diff
@@ -58,7 +58,7 @@ protected void subscribeActual(Subscriber<? super T> s) {
 
         Subscription s;
 
-        final SequentialDisposable timer = new SequentialDisposable();
+        Disposable timer;
```

Comment:
Timer is only manipulated from `onNext`, `onError` or `onComplete` and doesn't have to be atomically safe. The operator disposes the entire worker to stop any outstanding timer activity anyway.